sites/public/pages/account/dashboard.tsx

```diff
@@ -53,10 +53,6 @@ function Dashboard(props: DashboardProps) {
   return (
     <RequireLogin signInPath="/sign-in" signInMessage={t("t.loginIsRequired")}>
       <Layout>
-        <Head>
-          <title>{t("nav.myDashboard")}</title>
-        </Head>
-        <MetaTags title={t("nav.myDashboard")} description="" />
```

Comment:
Don't we want to keep the meta tags for the page still?

Reply:
Yes thank you! Sorry, I had moved them into the <Layout> component at one point 😅